war/src/main/webapp/images/build-status/build-status-sprite.svg

```diff
@@ -0,0 +1,61 @@
+<svg width="0" height="0" class="hidden">
```

Comment:
any reason this is using tabs and everything else spaces? and also others are minified but not this?

Reply:
Tabs are due because I copied & pasted svg onto it, I'll try to format it with spaces. The reason for this not being minified is because I'm manually editing it.